user_tools/docs/user-tools-onprem.md

```diff
@@ -0,0 +1,130 @@
+| **jvm_heap_size**              | The maximum heap size of the JVM in gigabytes                                                                                                                                                                                                                                                             | 24                                                                                                                                                   |     N    |
+| **tools_jar**                  | Path to a bundled jar including RAPIDS tool. The path is a local filesystem                                                                                                                                                                                                                               | Downloads the latest rapids-tools_*.jar from mvn repo                                                                                                |     N    |
+| **filter_apps**                | Filtering criteria of the applications listed in the final STDOUT table is one of the following (`NONE`, `SPEEDUPS`). "`NONE`" means no filter applied. "`SPEEDUPS`" lists all the apps that are either '_Recommended_', or '_Strongly Recommended_' based on speedups.                                   | `SPEEDUPS`                                                                                                                                           |     N    |
+| **verbose**                    | True or False to enable verbosity to the wrapper script                                                                                                                                                                                                                                                   | False if `RAPIDS_USER_TOOLS_LOG_DEBUG` is not set                                                                                                    |     N    |
+| **rapids_options****           | A list of valid [Qualification tool options](../../core/docs/spark-qualification-tool.md#qualification-tool-options). Note that (`output-directory`, `platform`) flags are ignored, and that multiple "spark-property" is not supported.                                                                  | N/A                                                                                                                                                  |     N    |
+
+#### Use case scenario
+
+A typical workflow to successfully run the `qualification` command in local mode is described as follows:
+
+1. Store the Apache Spark event logs in local directory.
+2. User sets up the development machine:
+   1. configures Java
```

Comment:
I know that this is the same line from other platforms. I just thought that the instruction is too vague.
We want to give some specific about what "configures java" mean. Or we replace it with something similar to  `on a machine with Java SDK 1.8+ installed1.